pkg/providers/v1/aws.go

```diff
@@ -5028,7 +5044,7 @@ func isFargateNode(nodeName string) bool {
 	return strings.HasPrefix(nodeName, fargateNodeNamePrefix)
 }
 
-func (c *Cloud) nodeNameToProviderID(nodeName types.NodeName) (InstanceID, error) {
+func (c *Cloud) nodeNameToInstanceID(nodeName types.NodeName) (InstanceID, error) {
```

Comment:
the provider id / instance id change is not super relevant, will put in separate commit . but I made it cuz strictly speaking all instance ids are provider ids but provider ids are not instance ids (since provider ids should have prefix aws://region or something) so this helped with m y understanding personally ...

Reply:
will fix unit tests due to rename as well